src/routes/templates/templates.json

```diff
@@ -3,7 +3,7 @@
     		"title": "svelte-app",
    		"url": "https://github.com/ln-dev7/userGithub",
     		"description": "web application that retrieves information from a GitHub profile",
-    		"npm": "",
+    		"npm": "npm run dev",
```

Comment:
The data behind his key is the package name, not the command to run.

For a template, the `npm` key don't have a lot of meaning thought